crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/controller/MeansAssessmentIntegrationTest.java

```diff
@@ -121,12 +121,12 @@ private MockHttpServletRequestBuilder buildRequest(HttpMethod method, String con
     }
 
     private void stubForRoleActionAndReservationUrl() {
-        stubFor(get(urlEqualTo("/api/internal/v1/assessment/role-action-url")).willReturn(aResponse()
+        stubFor(get(urlPathMatching("/api/internal/v1/assessment/authorization/users/([a-zA-Z]*)/actions/([a-zA-Z_]*)")).willReturn(aResponse()
```

Comment:
I'd stick these urls in a constant and keep using `urlEqualTo` where we're configuring the stubbing within a test.